README.md

```diff
@@ -1,38 +1,100 @@
+name: Lint CloudFormation Templates
+
+on: [push]
+
+jobs:
+  cloudformation-linter:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Cloud Formation Linter with Latest Version
+        uses: scottbrenner/cfn-lint-action@v2
+
+      - name: Print the Cloud Formation Linter Version & run Linter.
+        run: |
+          cfn-lint --version
+          cfn-lint -t ./template.yml
+```
+
+Within the `run |` section, you specify the required Cloud Formation Linter commands.
+
+However, if you would rather run the Cloud Cloud Formation Linter commands within this action. That is also possible. An example of that can be found below:
+
+```yaml
+name: Lint CloudFormation Templates
+
+on: [push]
+
+jobs:
+  cloudformation-linter:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Testing with CFN Lint Version & Command
+        uses: ./
+        with:
+          command: cfn-lint -t ./template.yml
+```
+
+Further, you can configure this action to download a specific version of the [CloudFormation Linter](https://github.com/aws-cloudformation/cfn-python-lint/), as well as the Python interpreter. See the table below for all the `INPUTS` this action can take.
+
+| Input Name | Input Description                                  | Default Value                                             | Required? |
+| ---------- | -------------------------------------------------- | --------------------------------------------------------- | --------- |
+| version    | Version of CFN PyPi Package                        | Latest Version of CFN PyPi Package                        | false     |
+| python     | Python Version                                     | Defaults to `python` on Windows, and `python3` otherwise. | false     |
+| command    | Cloud Formation Linter Command to Run Afer Install | N/A                                                       | false     |
+
+This GitHub Action does not directly output any values.
+
+## How to use the Cloud Formation Linter?
+
+See [Cloud Formation Linter Usage](https://github.com/aws-cloudformation/cfn-python-lint#basic-usage) for full usage details.
+
+## Upgrading from Version 1?
+
+The main difference between Version 1 and Version 2 is the fact Version 2 by default doesn't run any linting commands. The primary purpose of this action is now to provide the underlying [CloudFormation Linter](https://github.com/aws-cloudformation/cfn-python-lint/) package and enable you to run your commands. To do this, follow the steps below:
+
+#### Step One
+
+Change the action in your Workflow to be:
+
+```
+- name: Setup Cloud Formation Linter with Latest Version
+  uses: scottbrenner/cfn-lint-action@v2
+```
+
+#### Step Two
+
+Below the above include the following:
+
+```
+- name: Print the Cloud Formation Linter Version & run Linter.
+  run: |
+    cfn-lint --version
+    cfn-lint -t ./**/*.yaml
```

Comment:
Would it be possible to make `cfn-lint -t ./**/*.yaml` the default command for the Action, if no command is specified? Doing so might remove the "breaking change", in the situation where someone is currently using this Action as
```
- name: cfn-lint
      uses: docker://ghcr.io/scottbrenner/cfn-lint-action:master
      with:
        args: "**/*.yaml"
```
If it's not too much trouble, if the Action could continue to just run `cfn-lint -t ./**/*.yaml` by default, Workflows should continue to work just fine if I'm not mistaken.